__mocks__/react-native-permissions.ts

```diff
@@ -0,0 +1,77 @@
+import {PERMISSIONS, RESULTS} from 'react-native-permissions/dist/commonjs/permissions';
+import type {ValueOf} from 'type-fest';
+
+type Results = ValueOf<typeof RESULTS>;
+type ResultsCollection = Record<string, Results>;
+type NotificationSettings = Record<string, boolean>;
+type Notification = {status: Results; settings: typeof notificationSettings};
+
+const openLimitedPhotoLibraryPicker: jest.Mock<void> = jest.fn(() => {});
+const openSettings: jest.Mock<void> = jest.fn(() => {});
+const check: jest.Mock<Results> = jest.fn(() => RESULTS.GRANTED as string);
+const request: jest.Mock<Results> = jest.fn(() => RESULTS.GRANTED as string);
+const checkLocationAccuracy: jest.Mock<string> = jest.fn(() => 'full');
+const requestLocationAccuracy: jest.Mock<string> = jest.fn(() => 'full');
+
+const notificationOptions: string[] = ['alert', 'badge', 'sound', 'carPlay', 'criticalAlert', 'provisional'];
+
+const notificationSettings: NotificationSettings = {
+    alert: true,
+    badge: true,
+    sound: true,
+    carPlay: true,
+    criticalAlert: true,
+    provisional: true,
+    lockScreen: true,
+    notificationCenter: true,
+};
+
+const checkNotifications: jest.Mock<Notification> = jest.fn(() => ({
+    status: RESULTS.GRANTED,
+    settings: notificationSettings,
+}));
+
+const requestNotifications: jest.Mock<Notification> = jest.fn((options: Record<string, string>) => ({
+    status: RESULTS.GRANTED,
+    settings: Object.keys(options)
+        .filter((option: string) => notificationOptions.includes(option))
+        .reduce((acc: ResultsCollection, option: string) => ({...acc, [option]: true}), {
+            lockScreen: true,
+            notificationCenter: true,
+        }),
+}));
+
+const checkMultiple: jest.Mock<ResultsCollection> = jest.fn((permissions: string[]) =>
```

Comment:
Can we use `Permission` type from `react-native-permissions` here and in other places?
```suggestion
const checkMultiple: jest.Mock<ResultsCollection> = jest.fn((permissions: Permission[]) =>
```